samples/ukernel_mmt4d_riscv_32/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+iree_add_all_subdirs()
```

Comment:
not sure if you plan to land this or just treat it as an example PR, but if landing you'll want to predicate all this on whether the cpu backend is enabled and probably some other things - something like

```
iree_compiler_targeting_iree_arch(_IREE_COMPILER_TARGETING_RISCV_32 "riscv_32")
if(NOT IREE_TARGET_BACKEND_LLVM_CPU OR
   NOT _IREE_COMPILER_TARGETING_RISCV_32 OR
   NOT IREE_HAL_DRIVER_LOCAL_TASK OR
   NOT IREE_HAL_EXECUTABLE_LOADER_EMBEDDED_ELF)
  return()
endif()
```